main.py

```diff
@@ -40,311 +40,135 @@
 from inferno import *
 from scorescreen import *
 from howtoplay import *
```

Comment:
It'll be great to get rid of these imports and use relative/absolute imports.

Reply:
Yes, you can leave the flake8 warnings, I have covered most of them in the next commit, pushing them soon